specs/lightclient/beacon-chain.md

```diff
@@ -165,30 +186,59 @@ def process_block(state: BeaconState, block: BeaconBlock) -> None:
 ```python
 def process_sync_committee(state: BeaconState, block: BeaconBlock) -> None:
     # Verify sync committee aggregate signature signing over the previous slot block root
-    previous_slot = Slot(max(state.slot, 1) - 1)
+    previous_slot = Slot(max(int(state.slot), 1) - 1)
```

Comment:
@djrtwo 

This line was `previous_slot = max(state.slot, Slot(1)) - Slot(1)` earlier.

But since `max` needs to compare the same type, either we cast to `int` or cast to `int`.